tests/test_coils.py

```diff
@@ -214,9 +287,56 @@ def test_from_symmetry(self):
             coil, I, [0, 0, 1], np.pi / NFP, N // NFP // 2
         )
         coils2 = MixedCoilSet.from_symmetry(coils, NFP, True)
-        B_approx = coils2.compute_magnetic_field([10, 0, 0], basis="rpz", grid=32)[0]
+        B_approx = coils2.compute_magnetic_field(
+            [10, 0, 0], basis="rpz", source_grid=32
+        )[0]
         np.testing.assert_allclose(B_true, B_approx, rtol=1e-3, atol=1e-10)
 
+    @pytest.mark.unit
+    def test_symmetry_magnetic_field(self):
```

Comment:
Summary of what this new test is checking:

`sym_coilset` contains only 3 unique coils in half of one field period. It computes it's magnetic field assuming field period symmetry and stellarator symmetry, using the new code that was added. 

`asym_coilset` contains 24 coils but should be equivalent to the other coil set (3 unique coils, x2 for stellarator symmetry, x4 field periods). It computes it's magnetic field the "old school" way from each coil. 

This test checks that both coil sets give the same magnetic fields, in both coordinate systems.